### ISSUE TITLE
gallery instead of slider

### ISSUE THREAD

**Issue by sylvaincordiere** [mrleemon/gutenberg-slider#1]
Hi I install your plugin and what I get is more a gallery than a slider
![im](https://user-images.githubusercontent.com/39384609/42996303-711dca54-8c13-11e8-8084-ef10fe55f759.png)

**Comment by mrleemon** [comment]
Yep, sorry about that. The block is experimental and under development. I'm waiting for the next version of the Gutenberg plugin to arrive to fix this.

**Comment by sylvaincordiere** [comment]
you mean [gutenberg 3.3](https://github.com/WordPress/gutenberg/milestone/57) ?

**Comment by mrleemon** [comment]
Yes
